### ISSUE TITLE
Does not work with ASP.NET Core 1

### ISSUE THREAD

**Issue by rezamohamed** [madskristensen/WebEssentials2015#213]
None of the features work with ASP.NET Core 1 applications

**Comment by madskristensen** [comment]
Which features in particular? All the CSS and HTML editor features works just fine for example.

**Comment by rezamohamed** [comment]
Browserlink is what I use the most from WebEssentials 2015. This does not work - can you verify please?
Thanks!

**Comment by madskristensen** [comment]
Yes, that isn't working correctly do to issues with Browser Link in ASP.NET Core. I'm looking into it

**Comment by rezamohamed** [comment]
ZenCoding doesnt seem to work either.

**Comment by PoulErikDK** [comment]
Injected menu doesn't work - see attachment.
![2016-07-29_00-52-29](https://cloud.githubusercontent.com/assets/16957188/17232312/6c723852-5527-11e6-8551-56b4a8f99453.png)

**Comment by PoulErikDK** [comment]
Hi Mads,
When do you expect to fix Browser Link for .net core 1?
Regards,
Poul Erik

**Comment by PoulErikDK** [comment]
[09:49:14 GMT+0200 (Romance Summer Time)] Browser Link: Failed to invoke return value callback:
TypeError: Unable to get property 'files' of undefined or null reference
browserLink (64,455)
